workload/workload.py

```diff
@@ -0,0 +1,238 @@
+import json
+import subprocess
+
+from prettytable import PrettyTable
+
+test = False
+
+lfric_repositories = [
+    "lfric_apps",
+    "lfric_core",
+]
+
+ssd_repositories = [
+    "um",
+    "jules",
+    "socrates",
+    "casim",
+    "ukca",
+    "simulation-systems",
+    "SimSys_Scripts",
+    "git_playground",
+    "growss",
+]
+
+# Ideally get this list another way but for now hardcode it.
+other_reviewers = [
+    "MetBenjaminWent",
+    "jedbakerMO",
+    # Chris Maynard
+    # James CS
+    # Oakley Brunt
+]
+
+
+class ProjectData:
+    """
+    A class to hold GitHub project data. The focus is on review information.
+
+    data: dict Raw data from the project
+    review_data: list Data filtered to contain a list of review tuples
+    """
+
+    def __init__(self):
+        self.data = {}
+        self.review_data = []
+
+        self.fetch_project_data()
+        self.filter_reviewers()
+
+    def fetch_project_data(self):
+        """
+        Retrieve data from GitHub API or a from a test file.
+        """
+        if test:
+            with open("test.json") as f:
+                self.data = json.loads(f.read())
+
+        else:
+            command = "gh project item-list 376 -L 500 --owner MetOffice --format json"
+            output = subprocess.run(command, shell=True, capture_output=True)
```

Comment:
```suggestion
            output = subprocess.run(command.split(), capture_output=True, timeout=180)
```
Better to use `shell=False` if possible. Maybe worth raising an error here as well if an issue,

```
if output.rc:
    raise RuntimeError(Some log message+ output.stderr)
```

Reply:
done